src/main.rs

```diff
@@ -345,7 +349,11 @@ fn infer_batch_size(opts: &Opts, ulimit: u32) -> u16 {
             // ulimit is smaller than aveage batch size
             // user must have very small ulimit
             // decrease batch size to half of ulimit
-            warning!("Your file limit is very small, which negatively impacts RustScan's speed. Use the Docker image, or up the Ulimit with '--ulimit 5000'. ");
+            if !(cfg!(windows)) {
+                warning!("Your file limit is very small, which negatively impacts RustScan's speed. Use the Docker image, or up the Ulimit with '--ulimit 5000'. ");
+            } else {
+                warning!("Windows is known to be much slower than scanning on Unix systems. Use the Docker image if you need faster speeds.")
```

Comment:
This needs a semi-colon on the end

Reply:
🤦‍♀️ that's what i get for using the github editor smh